package.json

```diff
@@ -2,7 +2,7 @@
   "name": "open-lens",
   "productName": "OpenLens",
   "description": "OpenLens - Open Source IDE for Kubernetes",
-  "version": "5.0.0-beta.5",
+  "version": "5.0.0-beta.6",
```

Comment:
Did we skip `beta.5`? 🤔

Reply:
Yes, since I figured that the number just needs to be increasing. That is what the `make release-version prerelease` does.

Reply:
Is that fine? Version might be changed in non-release PR because of migrations.

Reply:
Yup, since `beta.5` is "before" `beta.6` all the migrations targeting upgrading from "before" `beta.5` to "after or equal to" `beta.5` will still run.

Reply:
I meant is it really fine to skip versions.